src/device_builder.rs

```diff
@@ -106,27 +107,60 @@ impl<'a, B: UsbBus> UsbDeviceBuilder<'a, B> {
         self
     }
 
+    /// Sets **extra** Language ID for device.
+    ///
+    /// Since "en_US"(0x0409) is implicitly embedded, you just need to fill other LangIDs
+    ///
+    /// Default: (none)
+    pub fn extra_lang_ids(mut self, extra_lang_ids: &'a [u16]) -> Self {
```

Comment:
Can we name this something more ergonomic, like `add_extra_languages()` or similar?

Reply:
Since this function override the extra language id list, I might think `set_extra_languages()` a better one?

Reply:
done